atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/ClientLogEvents.java

```diff
@@ -96,6 +97,33 @@ default CommitUpdate toCommitUpdate(LockWatchVersion startVersion, CommitInfo co
         return CommitUpdate.invalidateSome(locksTakenOut);
     }
 
+    default SpanningCommitUpdate toSpanningCommitUpdate(
```

Comment:
Is there a nice way to de-duplicate code here?

Reply:
I think both of the below suggestions are reasonable